examples/acquiring_data/querying_and_loading_SHARP_data.py

```diff
@@ -0,0 +1,38 @@
+"""
+===============================
+Querying and loading SHARP data
+===============================
+
+In this example we will demonstrate how to acquire [Spaceweather HMI Active Region Patch (SHARP)](http://jsoc.stanford.edu/doc/data/hmi/sharp/sharp.htm) data and load it into a `sunpy.map.Map`.
+"""
+import os
+
+import astropy.units as u
+
+import sunpy.map
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###################################################################
+# To get access to SHARP data, we will need to query the [JSOC/Stanford](http://jsoc.stanford.edu/).
+# We will use `Fido <sunpy.net.fido_factory.UnifiedDownloaderFactory>`
```

Comment:
Is this actually what we want to link to in our API docs?

Reply:
I thought this is how we link Fido, but is there another way?

Reply:
`sunpy.net.fido_factory.Fido` should work